asset_stream_manager/local_assets_stream_manager_service_impl.cc

```diff
@@ -17,21 +17,31 @@
 #include <iomanip>
 
 #include "absl/strings/str_format.h"
+#include "absl/strings/str_replace.h"
 #include "absl/strings/str_split.h"
 #include "asset_stream_manager/multi_session.h"
-#include "asset_stream_manager/session.h"
 #include "asset_stream_manager/session_manager.h"
 #include "common/grpc_status.h"
 #include "common/log.h"
 #include "common/path.h"
 #include "common/process.h"
 #include "common/sdk_util.h"
 #include "common/status.h"
+#include "google/protobuf/text_format.h"
 #include "manifest/manifest_updater.h"
 
+using TextFormat = google::protobuf::TextFormat;
+
 namespace cdc_ft {
 namespace {
 
+std::string RequestToString(const google::protobuf::Message& request) {
```

Comment:
Nit: Could have been `ProtoToString`, or `ToTextProto`.

Reply:
Ack. I slightly prefer RequestToString as that hints at how it's used.